test/markdown.jl

```diff
@@ -70,3 +70,9 @@ writemime(io::IO, m::MIME"text/plain", r::Reference) =
                                                           ["hgh",Bold("jhj"),"ge"],
                                                           "f"]],
                                                   [:l, :r, :r]))
+t = """a   |   b
```

Comment:
In retrospect I could've used `"""\` to start this on a new line to make this more readable (I'm not sure the \ is even necessary).